src/flamenco/runtime/program/fd_bpf_program_util.c

```diff
@@ -233,6 +236,7 @@ fd_bpf_scan_and_create_bpf_program_cache_entry( fd_exec_slot_ctx_t * slot_ctx,
 
     cnt++;
   }
+  FD_LOG_NOTICE(("VALUE: %lu", value));
```

Comment:
ditto

Reply:
These shouldn't have gone in... my mistake